### DISCUSSION TITLE
ESP32-S3-WROOM-2 with 1.8 V SPI Voltage

### DISCUSSION THREAD

**Discussion by helmutkohlkanzler** [hoylabs/OpenDTU-OnBattery#1133] [Q&A] [answered]
Hi all, has anybody tried the WROOM-2 with 1.8 V SPI voltage?
I ask because this ESP32 model is listed in the table (https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/ESP32-Versions-and-Memory). The adum1201 does not seem to support 1.8 V.
Any comments very welcome!
 - marcus

**Comment by spcqike** [comment, marked as answer]
Isn’t the SPI voltage only relevant for external flash memory? So in this case the 16Mb or 32Mb? 

I thought esp32 has 3.3V logic level voltage on its pins. 

See its data sheet 

<img width="375" alt="image" src="https://github.com/user-attachments/assets/8bdca1ca-f9f6-4b1e-8383-0a760a989575">
<img width="375" alt="image" src="https://github.com/user-attachments/assets/5805bb6a-047b-4270-96d5-826af9856bc8">

https://www.espressif.com/sites/default/files/documentation/esp32-s3-wroom-2_datasheet_en.pdf

**Reply by helmutkohlkanzler** [reply]
Thanks spcqike, as you quote only the GPIO47 and GPIO48 work on 1.8 V level. The GPIOs for general purpose SPI2 are GPIO10 ... GPIO12.
